### PR TITLE
fix 36

### PR DESCRIPTION
**Pull request by Grommers00**
I've updated it to include code that will get a list of your messages by user-name and message. 

It should only display locally, but will let you reference them.

### CODE REVIEW

File: tmessage/db.py [hunk 1]
```diff
@@ -17,11 +17,19 @@ class Meta:  # pylint: disable=missing-class-docstring,too-few-public-methods
         database = MESSAGES_DB
 
 
+
+@MESSAGES_DB
+def grab_messages(user):
+    """Grab messages from the user"""
+    cursor = MESSAGES_DB.execute_sql("SELECT sender, message FROM message where sender LIKE \'" + user +'\'')
```
[the remark this answers is not in the document]
Comment (Grommers00) [reply tmessage/db.py]: easier for a select statement I think to be in one line.

File: tmessage/db.py [hunk 2]
```diff
@@ -17,11 +17,19 @@ class Meta:  # pylint: disable=missing-class-docstring,too-few-public-methods
...
+    for row in cursor.fetchall():
+       print(row[0] + " : " + row[1])
+
```
[the remark this answers is not in the document]
Comment (Grommers00) [reply tmessage/db.py]: end of function

File: tmessage/db.py [hunk 3]
```diff
@@ -17,11 +17,19 @@ class Meta:  # pylint: disable=missing-class-docstring,too-few-public-methods
         database = MESSAGES_DB
 
 
+
+@MESSAGES_DB
+def grab_messages(user):
+    """Grab messages from the user"""
+    cursor = MESSAGES_DB.execute_sql("SELECT sender, message FROM message where sender LIKE \'" + user +'\'')
+    for row in cursor.fetchall():
+       print(row[0] + " : " + row[1])
```
[the remark this answers is not in the document]
Comment (Grommers00) [reply tmessage/db.py]: that is okay I think?

**Comment by Shanmugapriya03** [comment]
@Grommers00 Can you fix the linting issues as mentioned [here](https://travis-ci.com/Haider8/tmessage/builds/131871095)

**Comment by Grommers00** [comment]
wooo, i think I got it. Sorry first time using lint and python :)

**Comment by Haider8** [comment]
@Grommers00 Please describe the changes done by you briefly so I can test this PR.

File: tmessage/cli.py [hunk 4]
```diff
@@ -4,7 +4,7 @@
 import paho.mqtt.client as mqtt
 from colorama import init, deinit, Fore, Back, Style
 import tmessage.auth as auth  # auth.py
-from tmessage.db import store_messages  # db.py
+from db import grab_messages, store_messages  # db.py"""
```
Comment (Haider8) [inline tmessage/cli.py]: Make it

```suggestion
from tmessage.db import grab_messages, store_messages  # db.py"""
```

**Comment by Grommers00** [comment]
So what this function does if if you type 'grab' in the command line, it will grab a list of everything you've said previously.  I figured this would be an easy way for you to quickly grab a list of everything you've said that was stored in the database. I added a function that goes into the SQL database and grabs it all based on the author. Theoretically this would be the foundation to more code being added that would allow for you to grab on date, etc, or many other ways to grab conversations, there's alot of potential.

**Comment by Haider8** [comment]
@Grommers00 What command should I type exactly?

I am having this error right now on giving `grab` command
```
No command 'grab' found
```

**Comment by Grommers00** [comment]
Ah - are you logging in utilizing the --user "username"?

**Comment by Haider8** [comment]
@Grommers00 I appreciate the work done by you here. But what we want is a command like `tmessage --messages` which will show us all the messages which are stored in the database through paging. For paging, we can use this library [here](https://pypi.org/project/pager/).

**Pull request closed without merging**